### PR TITLE
cleanups and fixes

### PR DESCRIPTION
**Pull request by bomsy**
Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* Added scrolling of search list
* Show just the relative url i.e remove domains
* Show file icons for the file paths
* add test for `getRelativePath`
* Change the font for results

### Screenshots/Videos (OPTIONAL)

image

### CODE REVIEW

File: src/components/ProjectSearch/TextSearch.js [hunk 1]
```diff
@@ -68,7 +77,7 @@ export default class TextSearch extends Component {
     return dom.div(
       {
         className: classnames("result", { focused }),
-        onClick: () => this.selectMatchItem(match)
+        onClick: () => setTimeout(() => this.selectMatchItem(match), 50)
```
Comment (jasonLaster) [inline src/components/ProjectSearch/TextSearch.js]: why is the timeout necessary?

File: src/components/ProjectSearch/TextSearch.js [hunk 2]
```diff
@@ -56,7 +64,8 @@ export default class TextSearch extends Component {
           expanded
         })
       }),
-      dom.span({ className: "file-path" }, file.filepath),
+      Svg("file"),
```
Comment (jasonLaster) [inline src/components/ProjectSearch/TextSearch.js]: 👍

File: src/utils/sources-tree.js [hunk 3]
```diff
@@ -85,6 +85,11 @@ function getFilenameFromPath(pathname?: string) {
   return filename;
 }
 
+function getRelativePath(path: string) {
+  const re = /(http(?:s?):\/\/(?:www\.)?[a-z0-9\-.]+)\/(.*)/i;
+  return path.match(re)[2];
+}
```
Comment (jasonLaster) [inline src/utils/sources-tree.js]: oh, surprised we didnt have this

File: src/components/ProjectSearch/TextSearch.js (hunk 1 above)
Comment (bomsy) [inline src/components/ProjectSearch/TextSearch.js]: On clicking a match, the page opens so quickly you have no visual indication of what you clicked. This just allows, the user see a visual indication for a few seconds before the source file is opened

**Comment by jasonLaster** [comment]
Love it

**Comment by codehag** [comment]
removed yarn.lock commit

**Pull request merged**